Updates

Mikle103 opened the pull request:
Have updated game file in rules and infamy, skills and Experiacne
created Chinese Tong catalog

updated following catalogs with infamy/skills/Exp.
Cowboys, Lawmen, Outlaws, US Cavalry, Buffalo Soldiers and Chinese Tong.

Mikle103 committed:
Update errors

Mikle103 committed:
Character cat

Mikle103 committed:
new posse and infamy updates

Mikle103 committed:
updates

Mikle103 committed:
updated infamy/skills/experiance

Mikle103 committed:
updates infamy/skills/exp

Mikle103 committed:
update skills/exp

Mikle103 committed:
updates skills/infamy/exp

Mikle103 committed:
Update infamy/skills/exp

Mikle103 committed:
Updated skills/infamy/exp

Mikle103 committed:
Update cat with skills /infamy/ exp

Mikle103 committed:
Moved 'The Kids done Good' to gst file + update cats

Mikle103 committed:
The Kids done Good update

Mikle103 committed:
The Kid done Good Update

Mikle103 committed:
Character added and updated complete from Rule Book.

Mikle103 commented:
Hey Could you please review and merge this data into main repo.

Cheers

Mikle103 committed:
All Native Tribes Updated

Mikle103 committed:
Frontier Hired Guns + rules/item updated gst file

Atr0k commented:
I've been reviewing the pull request... for some reason, looks like all IDs across all catalogs have been regenerated (so it would break any existing roster). Is that correct?

Mikle103 commented:
Not sure about breaking existing rosters?
I was do not know how the IDs were change as just edited or created files?
sorry

Mikle103 committed:
Hired guns / Legends from the Frontier Book

Mikle103 commented:
Have been thinking about the ID change and breaking existing rosters,
I have come to the conclusion yes all old rosters will break as when I updated then catalogs, I moved some entries and put links to them, this was for the core models.
Sorry

Mikle103 committed:
Last of the posses i thinks and some more characters

amis92 reviewed (changes requested):
I think your update could be merged. Do you think you could move over the old IDs from the current version?

Mikle103 commented:
@amis92 how do i go about this? unsure on how to do this?

amis92 commented:
Reading this again, it seems that you've replaced Root Entries with links to Shared Entries. If this is the case, nothing can be done to not-break rosters. Can you confirm this is the case?

Mikle103 commented:
@amis92 
Hi yes that is what I have done it helped with validations

amis92 commented:
Well then I think it's a big enough improvement to introduce these unavoidable breaking changes. Approved.
